modules/serverless-logic-running-workflows.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-running-workflows"]
+= Running a workflow locally
+
+You can use the `run` command with `kn workflow` to build and run your {ServerlessLogicProductName} workflow project in your current directory.
+
+
+.Prerequisites
+
+* The {ServerlessLogicOperatorName} is installed on the cluster.
```

Comment:
{ServerlessLogicOperatorName} will not reflect right now because I have added this attribute in the (https://github.com/openshift/openshift-docs/pull/76488). Once we merge that PR this attribute will reflect correctly.